app/models/package_manager/multiple_sources_base/provider_info.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module PackageManager
+  class MultipleSourcesBase < Base
+    class ProviderInfo
+      attr_reader :identifier, :provider_class
+
+      def initialize(identifier:, provider_class:, default: false)
```

Comment:
Should this include something about which ecosystem they are a provider for? e.g. 'Pypi' or 'Rubygems'?

Reply:
No, that's not information needed for these objects.